packages/flutter/test/widgets/scroll_view_test.dart

```diff
@@ -410,4 +411,172 @@ void main() {
     await tester.dragFrom(const Offset(100.0, 100.0), const Offset(0.0, 100.0));
     expect(scrolled, isFalse);
   });
+
+  testWidgets('separatorBuilder must return something', (WidgetTester tester) async {
+    final List<String> listOfValues = <String>['ALPHA', 'BETA', 'GAMMA', 'DELTA'];
+
+    Widget buildFrame(Widget firstSeparator) {
+      return MaterialApp(
+        home: Material(
+          child: ListView.separated(
+            itemBuilder: (BuildContext context, int index) {
+              return ListTile(
```

Comment:
For tests, you don't really need to create a ListTile, just a Text widget will be OK